pontoon/insights/templates/insights/widgets/insights.html

```diff
@@ -51,9 +51,9 @@ <h3 class="controls">
         <section class="half">
             <div class="block clearfix">
                 <h3 class="controls">
-                    Unreviewed suggestions lifespan
+                    Age of unreviewed suggestions
                     {{ Tooltip.display(
-                        intro='How much time it takes on average to review a suggestion.',
+                        intro='How old are the unreviewed suggestions on average in particular point in time.',
```

Comment:
This sounds a bit off to me, I think is the more common expression "at any particular point in time", or "at a particular point in time"?